TouchFree/Assets/TouchFree/Scripts/CallToInteract/CallToInteractController.cs

```diff
@@ -103,121 +72,109 @@ void OnDisable()
             Destroy(CTIImage.texture);
             CTIImage.texture = null;
         }
+
+        StopAllCoroutines();
+        delayedSetupCoroutine = null;
+        showAfterHandsLostCoroutine = null;
     }
 
-    float setupCooldown = 0.5f;
     /// <summary>
-    /// This is provided on a cooldown to prevent the loading of video assets on every update of settings values
+    /// 
```

Comment:
Missing description in this comment, also we're using the XML-style comments which is inconsistent with things we do elsewhere in this repo. Do we wanna commit to this style for TF?

Reply:
This summary really only exists to explain the functionality of the _immediate param, so perhaps I just have it as a simple comment as opposed to a summary?